configs/default/SPBE-SPEEDYBEEF405V3.config

```diff
@@ -0,0 +1,134 @@
+# Betaflight / STM32F405 (S405) 4.3.0 Jun 14 2022 / 00:47:24 (229ac66) MSP API: 1.44
+
+board_name SPEEDYBEEF405V3
+manufacturer_id SPBE
+
+# resources
+resource BEEPER 1 C05
+resource MOTOR 1 B06
+resource MOTOR 2 B07
+resource MOTOR 3 B08
+resource MOTOR 4 B09
+resource MOTOR 5 B00
+resource MOTOR 6 B01
+resource MOTOR 7 B05
+resource MOTOR 8 B04
+resource PPM 1 A03
+resource PWM 1 A08
+resource LED_STRIP 1 C09
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 2 A02
+resource SERIAL_TX 3 C10
+resource SERIAL_TX 4 A00
+resource SERIAL_TX 5 C12
+resource SERIAL_TX 6 C06
+resource SERIAL_RX 1 A10
+resource SERIAL_RX 2 A03
+resource SERIAL_RX 3 C11
+resource SERIAL_RX 4 A01
+resource SERIAL_RX 5 D02
+resource SERIAL_RX 6 C07
+resource I2C_SCL 2 B10
+resource I2C_SDA 2 B11
+resource LED 1 C08
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_MISO 1 A06
+resource SPI_MISO 2 B14
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 2 B15
+resource CAMERA_CONTROL 1 B03
+resource ADC_BATT 1 C00
+resource ADC_RSSI 1 C02
+resource ADC_CURR 1 C01
+resource SDCARD_CS 1 A15
+resource PINIO 1 C03
+resource OSD_CS 1 B12
+resource GYRO_EXTI 1 C04
+resource GYRO_CS 1 A04
+
+# timer
+timer B06 AF2
+# pin B06: TIM4 CH1 (AF2)
+timer B07 AF2
+# pin B07: TIM4 CH2 (AF2)
+timer B08 AF2
+# pin B08: TIM4 CH3 (AF2)
+timer B09 AF2
+# pin B09: TIM4 CH4 (AF2)
+timer B00 AF2
+# pin B00: TIM3 CH3 (AF2)
+timer B01 AF2
+# pin B01: TIM3 CH4 (AF2)
+timer B05 AF2
+# pin B05: TIM3 CH2 (AF2)
+timer B04 AF2
+# pin B04: TIM3 CH1 (AF2)
+timer A08 AF1
+# pin A08: TIM1 CH1 (AF1)
+timer A03 AF2
+# pin A03: TIM5 CH4 (AF2)
+timer B03 AF1
+# pin B03: TIM2 CH2 (AF1)
+timer C09 AF3
+# pin C09: TIM8 CH4 (AF3)
+
+# dma
+dma ADC 1 0
+# ADC 1: DMA2 Stream 0 Channel 0
+dma pin B06 0
+# pin B06: DMA1 Stream 0 Channel 2
+dma pin B07 0
+# pin B07: DMA1 Stream 3 Channel 2
+dma pin B08 0
+# pin B08: DMA1 Stream 7 Channel 2
+dma pin B00 0
+# pin B00: DMA1 Stream 7 Channel 5
+dma pin B01 0
+# pin B01: DMA1 Stream 2 Channel 5
+dma pin B05 0
+# pin B05: DMA1 Stream 5 Channel 5
+dma pin B04 0
+# pin B04: DMA1 Stream 4 Channel 5
+dma pin A08 0
+# pin A08: DMA2 Stream 6 Channel 0
+dma pin A03 0
+# pin A03: DMA1 Stream 1 Channel 6
+dma pin B03 0
+# pin B03: DMA1 Stream 6 Channel 3
+dma pin C09 0
+# pin C09: DMA2 Stream 7 Channel 7
+
+# feature
+feature -RX_PARALLEL_PWM
```

Comment:
please remove as RX_SERIAL is now default

Reply:
thank you edited